pySDC/core/__init__.py

```diff
@@ -1 +1,5 @@
+from pySDC.core.Nodes import NodesGenerator
+from pySDC.core.Lagrange import LagrangeApproximation
+
 __author__ = 'robert'
+__all__ = ['NodesGenerator', 'LagrangeApproximation']
```

Comment:
Do you really need this somewhere? Importing `all` is bad practice anyway.

Reply:
I'm not so sure why this came up ... I'll remove it

Reply:
Oh, actually I know why : this is because of the first two lines before :

```python
from pySDC.core.Nodes import NodesGenerator
from pySDC.core.Lagrange import LagrangeApproximation
```

Those allow more convenient import statements

```
from pySDC.core import LagrangeApproximation, NodesGenerator
```
but then we have to add the `__all__` variable to avoid this kind of flake8 errors :
```
pySDC/core/__init__.py:1:1: F401 'pySDC.core.Nodes.NodesGenerator' imported but unused
from pySDC.core.Nodes import NodesGenerator
^
pySDC/core/__init__.py:2:1: F401 'pySDC.core.Lagrange.LagrangeApproximation' imported but unused
from pySDC.core.Lagrange import LagrangeApproximation
^
2     F401 'pySDC.core.Nodes.NodesGenerator' imported but unused
2
```

Reply:
OK, good.